src/common/components/SubscribeChannelDropdown.tsx

```diff
@@ -0,0 +1,33 @@
+import OptinNotifSettingDropdown from 'components/dropdowns/OptinNotifSettingDropdown';
+import { ChannelDetailsProps } from 'modules/dashboard/configs/DashboardFeaturedChannels.config';
+import { ChannelDetailsResponse } from 'queries';
+import { FC, ReactNode } from 'react';
+
+interface SubscribeChannelDropdownProps {
+  children: ReactNode;
+  channelDetails: ChannelDetailsResponse;
+  setSubscribed: React.Dispatch<React.SetStateAction<boolean>>;
+  setSubscriberCount: React.Dispatch<React.SetStateAction<number>>;
+  setLoading: (value: boolean) => void;
+}
+
+const SubscribeChannelDropdown: FC<SubscribeChannelDropdownProps> = (options) => {
```

Comment:
ser I thought that in the future it will help us in refractoring the OptinNotifSettingDropdown component and it will separate the old and new components.